sig_tracker/sig_tracker/urls.py

```diff
@@ -1,8 +1,7 @@
-
 from django.contrib import admin
 from django.conf.urls import include,url
 
 urlpatterns = [
-    url(r'admin/', admin.site.urls),
-    url(r'about_sig/', include('about_sig.urls')),
+    url(r'^admin/', admin.site.urls),
```

Comment:
I don't think this URL would match any pattern since the URL to match is `http://127.0.0.1:8000/admin`. Please check _all_ changes locally!